kernel/init.c

```diff
@@ -373,6 +436,10 @@ FUNC_NORETURN void _Cstart(void)
 	__stack_chk_guard = (void *)sys_rand32_get();
 #endif
 
+#ifdef CONFIG_SMP
+	smp_init();
+#endif
+
 	/* display boot banner */
```

Comment:
Minor: It would be nice to have "SMP" somewhere in the boot banner.

Reply:
Good idea.